docker-compose.yml

```diff
@@ -7,15 +7,20 @@ services:
       - "16385:6379"
       - "6379:6379"
 
+  redis8:
+    image: redis:8.0
+    ports:
+      - "16388:6379"
+
   redis7-tls:
     image: redis:7.4
     volumes:
       - ./contrib/tls:/tls
     command: |
-      redis-server --enable-debug-command yes --protected-mode no 
-      --tls-port 6379 --port 0 
-      --tls-cert-file     /tls/redis.crt 
-      --tls-key-file      /tls/redis.key 
+      redis-server --enable-debug-command yes --protected-mode no
```

Comment:
There were quite a few trailing spaces in this file, which is not great and might create unexpected issues in some cases.

My IDE automatically fixed it, hope it's ok. I can revert these changes if needed.

Reply:
Thanks for cleaning this up!